bytewriter/memorymappedwriter.go

```diff
@@ -3,8 +3,8 @@ package bytewriter
 import (
 	"fmt"
 	"os"
-	"path"
-	"syscall"
+	path "path/filepath"
```

Comment:
Don't do this. In go, path and filepath are separate packages and the distinction is important in code. Instead replace all occurrences of `path` with `filepath`